py/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+
+- Move json calls to parameters. ([#3480](https://github.com/getsentry/relay/pull/3480))
```

Comment:
_nit_: Without reading the code, it's hard to understand what the change here actually is or how I'd use the new parameters.